<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertInfoDlg</class>
 <widget class="QDialog" name="CertInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Certificate information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mKIDText">
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>KID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Certificate </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mInfoTab">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QPlainTextEdit" name="mDetailText">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>221</red>
               <green>221</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>221</red>
               <green>221</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>240</red>
               <green>240</green>
               <blue>240</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Detail Value</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="mFieldTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Field List</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="mFieldTypeCombo"/>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mPathTab">
      <attribute name="title">
       <string>Path Validataion</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Certificate Path Tree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="mCertTree">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Path Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mCertLogText">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>221</red>
               <green>221</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>221</red>
               <green>221</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mManGroup">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="3" column="0">
             <widget class="QPushButton" name="mSaveToCABtn">
              <property name="text">
               <string>Save to CA</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="mCRLCheckBtn">
              <property name="toolTip">
               <string>Check revoked certificate with CRL</string>
              </property>
              <property name="text">
               <string>CRL Check</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="mGetCABtn">
              <property name="toolTip">
               <string>Get CA certificate and View certificate information</string>
              </property>
              <property name="text">
               <string>Get CA</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="mSaveToManBtn">
              <property name="text">
               <string>Save to Man</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="mPathValidBtn">
              <property name="toolTip">
               <string comment="Certificate path tree validation"/>
              </property>
              <property name="text">
               <string>Path Validation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="mMakeTreeBtn">
              <property name="toolTip">
               <string>Make certificate path tree</string>
              </property>
              <property name="text">
               <string>Make Tree</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="mSaveTrustedCABtn">
              <property name="text">
               <string>Save To RCA</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="mOCSPClientBtn">
              <property name="text">
               <string>OCSP Client</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="mGetCA_BERBtn">
              <property name="text">
               <string>Get CA BER</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="mDecodeCertBtn">
              <property name="toolTip">
               <string>Decode certificate in editor</string>
              </property>
              <property name="text">
               <string>Decode Cert</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="mVerifyCertBtn">
              <property name="toolTip">
               <string>Verify certificate signature</string>
              </property>
              <property name="text">
               <string>Verify Cert</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="mOCSPCheckBtn">
              <property name="toolTip">
               <string>Ceritificattion valid check with OCSP Server</string>
              </property>
              <property name="text">
               <string>OCSP Check</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="mGetCRL_BERBtn">
              <property name="text">
               <string>Get CRL BER</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="mGetCRLBtn">
              <property name="toolTip">
               <string>Get CRL and View CRL Information</string>
              </property>
              <property name="text">
               <string>Get CRL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mSaveBtn">
       <property name="toolTip">
        <string>Save certificate as pem</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mViewPubKeyBtn">
       <property name="text">
        <string>View PubKey</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mFieldTypeCombo</tabstop>
  <tabstop>mFieldTable</tabstop>
  <tabstop>mDetailText</tabstop>
  <tabstop>mCertTree</tabstop>
  <tabstop>mCertLogText</tabstop>
  <tabstop>mMakeTreeBtn</tabstop>
  <tabstop>mVerifyCertBtn</tabstop>
  <tabstop>mDecodeCertBtn</tabstop>
  <tabstop>mPathValidBtn</tabstop>
  <tabstop>mGetCABtn</tabstop>
  <tabstop>mGetCA_BERBtn</tabstop>
  <tabstop>mCRLCheckBtn</tabstop>
  <tabstop>mGetCRLBtn</tabstop>
  <tabstop>mGetCRL_BERBtn</tabstop>
  <tabstop>mOCSPCheckBtn</tabstop>
  <tabstop>mOCSPClientBtn</tabstop>
  <tabstop>mSaveToManBtn</tabstop>
  <tabstop>mSaveToCABtn</tabstop>
  <tabstop>mSaveTrustedCABtn</tabstop>
  <tabstop>mSaveBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
